blocks/commerce-company-structure/README.md

```diff
@@ -0,0 +1,92 @@
+# Commerce Company Structure Block
+
+## Overview
+
+The Commerce Company Structure block displays company organizational structure and management interface for B2B customers. It integrates with the Adobe Commerce Company Management dropin to show company hierarchy, manage company users, and provide administrative controls for company structure.
+
+## Integration
+
+### Block Configuration
+
+This block does not use `readBlockConfig()` - all configuration is handled through global commerce configuration and dropin initialization.
+
+### URL Parameters
+
+- No URL query parameters are used by this block
+
+### Local Storage
+
+- No localStorage keys are directly used by this block (authentication state is managed by commerce framework)
+
+### Events
+
+#### Event Listeners
+
+- Authentication state changes are handled through the commerce framework
+- Company data updates are managed by the dropin
+
+#### Event Emitters
+
+- `events.emit('company/structure-loaded', data)` - Emitted when company structure data is loaded
+- `events.emit('company/user-updated', user)` - Emitted when company user information is updated
```

Comment:
If these are emitted from within the dropin, they don't need to be noted. More specifically, we don't want to be documenting things that are hard to discover, just specifically the block code as-is.

Reply:
Updated the section